packages/ember-data/lib/system/snapshot.js

```diff
@@ -0,0 +1,299 @@
+/**
+  @module ember-data
+*/
+
+var get = Ember.get;
+
+var hasPropertyAccessors = Ember.__loader.require('ember-metal/platform')['hasPropertyAccessors'];
```

Comment:
This are exported by Ember as `Ember.platform.hasPropertyAccessors ` since at least 1.8.

https://github.com/emberjs/ember.js/blob/v1.8.1/packages/ember-metal/lib/main.js#L183-L186